x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_response_extension.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import styled from 'styled-components';
+import { PackagePolicyResponseExtensionComponentProps } from '@kbn/fleet-plugin/public';
+
+import { useIsExperimentalFeatureEnabled } from '../../../../../common/hooks/use_experimental_features';
+import { PolicyResponseWrapper } from '../../../../components/policy_response';
+
+const Container = styled.div`
+  padding: ${({ theme }) => theme.eui.paddingSizes.m};
+`;
+
+/**
+ * Exports Endpoint-specific package policy response
+ */
+export const EndpointPolicyResponseExtension = memo<PackagePolicyResponseExtensionComponentProps>(
+  ({ endpointId }) => {
+    const isPolicyResponseInFleetEnabled = useIsExperimentalFeatureEnabled(
+      'policyResponseInFleetEnabled'
+    );
+    if (!isPolicyResponseInFleetEnabled) return null;
```

Comment:
Do you need to check FF here? 

What happens in Fleet when the FF is off? will the flyout still be shown because it sees that the package has an extension registered and just display nothing?  _(the GIF in the description does not show the Fleet side of things when FF is off, so I can't really tell)_

My suggestion would be to move the check for the FF to the `plugin.tsx` and only register it if the FF is `true`.

Reply:
Actually it renders nothing, so the behaviour is the same as before when FF is disabled. I will move the FF check to the plugin.tsx, I think it makes more sense

Reply:
ahh. Ok. Did not realize that there was already content there in a accordion:

<img width="623" alt="image" src="https://user-images.githubusercontent.com/56442535/169528881-029c2f8d-f5e2-4bc9-9452-5bd828392fd3.png">


At one point I think it was just a link to the integration policy details page.

Sorry for the noise.

Reply:
Let me know if this looks good: https://github.com/elastic/kibana/pull/132312/commits/a60081b221cb099c443c65f460622889ca3af51f